bundles/org.openhab.binding.daikin/src/test/java/org/openhab/binding/daikin/internal/api/AirbaseControlInfoTest.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.daikin.internal.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openhab.binding.daikin.internal.api.airbase.AirbaseControlInfo;
+import org.openhab.binding.daikin.internal.api.airbase.AirbaseEnums.AirbaseFanMovement;
+
+/**
+ * This class provides tests for deconz lights
+ *
+ * @author Leo Siepel - Initial contribution
+ *
+ */
+
+@NonNullByDefault
+public class AirbaseControlInfoTest {
```

Comment:
Again, the f_dir_ud and f_dir_lr do not apply to Airbase.

Reply:
moved.